kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestMetricsContext.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest;
+
+import io.confluent.rest.metrics.RestMetricsContext;
+import org.apache.kafka.common.utils.AppInfoParser;
+
+import java.util.Map;
+
+public final class KafkaRestMetricsContext {
```

Comment:
Should this implement `MetricsContext`?

Reply:
Since KafkaRest itsself does not manage any metrics reporters and we can only propagate metrics contexts through configs it seemed appropriate to just expose the MetricsContext as opposed to implementing the interface.